Makefile

```diff
@@ -32,6 +32,20 @@ package: build ## Packages app
 	mv tmp/client dist
 	rmdir tmp
 
+package-ci:
```

Comment:
Worth documenting why this different from `make package` for future readers?

Reply:
done